users/management/commands/verify_user_education.py

```diff
@@ -0,0 +1,54 @@
+from django.core.management.base import BaseCommand
+import requests
+from django.contrib.auth import get_user_model
+from users.models import EducationInformation
+from django.core.exceptions import ObjectDoesNotExist
+from users.serializers import UserEducationInformationSerializer
+User = get_user_model()
+
+class Command(BaseCommand):
+
+    help = "Verify User Education"
+
+
+    def add_arguments(self, parser):
+        parser.add_argument('id', type=int, nargs='?', help='Define a user id')
+        parser.add_argument('-a', '--all', type=bool, nargs='?', default=False, help="Get all users education status")
+
+    def handle(self, *args, **options):
+        is_all = options['all']
+        user_id = options['id']
+        degree_ids = list()
+        if is_all:
+            users = User.objects.prefetch_related('educations')
+            for user in users:
+                user_education_info = user.educations.order_by('-degree_completed_date').first()
+                if user_education_info:
+                    context = UserEducationInformationSerializer(user_education_info).data
+                    response = self.verify_degree_api(user.username, **context).json()
```

Comment:
There is a lot of code duplication in this block and the block below. We should implement DRY here.

Reply:
Done.